packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -383,6 +384,10 @@ export default class XdslAccessCtrl {
               this.$scope.access.xdsl = assign(access, {
                 isFiber: includes(this.PACK.fiberAccess, access.accessType),
               });
+              this.$scope.access.xdsl.isNotSdsl = true;
```

Comment:
Nitpicking: We can directly write
```suggestion
              this.$scope.access.xdsl.isNotSdsl = this.$scope.access.xdsl.accessType !== 'sdsl' ;
```